content/automate/ManageIQ/System/Event/EmsEvent/RHEVM.class/__methods__/update_vm_import_status.rb

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
@tinaafitz thank you very much for your help! With this example I've been able to write the tests for this method, posted new version.